receiver/sqlqueryreceiver/README.md

```diff
@@ -22,8 +23,8 @@ A _query_ consists of a sql statement and one or more _metrics_, where each metr
 Each _metric_ in the configuration will produce one OTel metric per row returned from its sql query.
 
 * `metric_name`(required): the name assigned to the OTel metric.
-* `value_column`(required): the column name in the returned dataset used to set the value of the metric's datapoint. The column's values must be of an integer type.
-* `attribute_columns`(optional): a list of column names in the returned dataset used to set attibutes on the datapoint.
+* `value_column`(required): the column name in the returned dataset used to set the value of the metric's datapoint. The column's values must be of an integer type. This may be case-sensitive, depending on the driver (i.e. Oracle DB).
```

Comment:
I just noticed this sentence I added that is no longer true. Mind deleting it while you're in here?
```suggestion
* `value_column`(required): the column name in the returned dataset used to set the value of the metric's datapoint. This may be case-sensitive, depending on the driver (i.e. Oracle DB).
```

Reply:
Also, nit, but I believe `i.e.` is generally accepted to mean "that is" or "in other words". So I think you'll want this to be something like `e.g.`. Here and on line 27. 😄

Reply:
Good catch, I'll update.